.github/workflows/run_tests.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Run tests on all platforms
+name: CI/CD
```

Comment:
Reverting this

```suggestion
name: Run tests on all platforms
```

to keep the records in https://github.com/ivadomed/ivadomed/actions more consistent (and because it's blocking the final merge)

Signed-off-by: @dyt811

Reply:
I'm not sure if is it making a difference actually 🤔

Reply:
You're right, it wasn't. The Checks were stuck at yellow because they were expecting these

![Screenshot 2022-06-02 at 03-17-03 Branch protection rule · ivadomed_ivadomed](https://user-images.githubusercontent.com/987487/171574883-a062be06-fe8d-4707-a87c-5a55651027a7.png)

Sorry.

There's actually several things in CI that I noticed that would be worth fixing while doing this. So submit the name changes again in a smaller patch that's easier to review and I'll do the same for the things I noticed :)